ingestion_server/ingestion_server/slack.py

```diff
@@ -61,14 +61,17 @@ def _message(text: str, summary: str = None, level: Level = Level.INFO) -> None:
 
 
 def verbose(text: str, summary: str = None) -> None:
+    log.debug(text)
```

Comment:
This (and all other `log` calls in made by this PR) should have [`stacklevel=2`](https://docs.python.org/3/library/logging.html#logging.Logger.debug) added to them:

```suggestion
    log.debug(text, stacklevel=2)
```

This will ensure that when a log line is emitted from this module, it appears in the logs as if it came from the place where `slack.verbose` (or the other logging functions) were called. So rather than something like

```
DEBUG slack.py:63 - <the real message>
```

it'd say something like

```
DEBUG - indexer.py:102 - <the real message>
```

Reply:
OK